### PR TITLE
doc: Update Vulnerability Response Process 

### PR DESCRIPTION
**Pull request by RoboticMind**
waiting on confirmation and information about the @gridcoin.us email on before committing, but wanted to open this to feedback  while we wait

* Rename the file to SECURITY.md to work with github's security tab 
* Remove roman numerals and mention sections by name
   * Easier reading since you don't have to look at what things like section III are 
* Update information
  * Security Response Team Members
  * Cyptocurrencytalk mentions -> Discord since it's probably our largest platform
  * Update and add links and fix the broken slack link

### CODE REVIEW

**Review by jamescowens — COMMENTED** [review]
I am not sure in III. 3. whether we really want to require response via encrypted, secure channels. That actually might delay/harm communication... Thoughts?

**Comment by div72** [comment]
> Everything else is noise.

[SECURITY.MD of bitcoin](https://github.com/bitcoin/bitcoin/blob/master/SECURITY.md) is much simpler.

**Comment by jamescowens** [comment]
I actually prefer Bitcoin's simpler approach. My gpg fingerprint is 2F03 F85D 0DD8 6635 7698  4E17 2804 081B E891 286A.

**Comment by nathanielcwm** [comment]
Github provides a security template.

It appears like bitcoin based their security response on the template.

```
# Security Policy

## Supported Versions

Use this section to tell people about which versions of your project are
currently being supported with security updates.

| Version | Supported          |
| ------- | ------------------ |
| 5.1.x   | :white_check_mark: |
| 5.0.x   | :x:                |
| 4.0.x   | :white_check_mark: |
| < 4.0   | :x:                |

## Reporting a Vulnerability

Use this section to tell people how to report a vulnerability.

Tell them where to go, how often they can expect to get an update on a
reported vulnerability, what to expect if the vulnerability is accepted or
declined, etc.
```

**Comment by nathanielcwm** [comment]
It's not crypto related but the [SECURITY.MD](https://github.com/microsoft/PowerToys/blob/master/SECURITY.md) for Microsoft powertools has more detail than the Bitcoin one while still remaining quite brief.

I believe a language preference section would be a good idea.

**Comment by div72** [comment]
> I believe a language preference section would be a good idea.

I really don't think that's necesarry.

**Comment by jamescowens** [comment]
I want to go with Bitcoin's approach. Simpler is better. Please make the change.

**Comment by RoboticMind** [comment]
Alright will do that soon

**Comment by jamescowens** [comment]
@cyrossignol Please provide a fingerprint from a gpg key and then we can finalize this.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Reporting a Vulnerability
+
+Contact [security@gridcoin.us](mailto:security@gridcoin.us) to help notify developers about security issues
+
+For sensitive information, the following keys can be used:
+
+|Developer | Fingerprint |
+|-|-|
+|[Cy Rossignol](https://github.com/cyrossignol) (also known as cycy) |  |
```
Comment (cyrossignol) [inline SECURITY.md]:
```suggestion
|[Cy Rossignol](https://github.com/cyrossignol) (also known as cycy) | 87AC 53BC B65E B077 DEE2 0762 EBDC 4206 A699 78EC |
```

**Comment by jamescowens** [comment]
I am ready to merge this. @RoboticMind if you are good, please mark it ready for review.

**Review by jamescowens — APPROVED** [review]

**Comment by jamescowens** [comment]
@RoboticMind can you squash this down to two commits... the name change and then everything else in the second commit?

**Comment by RoboticMind** [comment]
Done! Now there should be just two commits. I meant to do that earlier, but I forgot about it

**Pull request merged**